app/controllers/SearchAuthoritiesController.scala

```diff
@@ -25,16 +25,16 @@ import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 import javax.inject.Inject
 import scala.concurrent.ExecutionContext
 
-class SearchAuthoritiesController @Inject()(
-                                             acc40Connector: Acc40Connector,
-                                             cc: ControllerComponents
+class SearchAuthoritiesController @Inject()(acc40Connector: Acc40Connector,
+                                            cc: ControllerComponents
                                            )(implicit ec: ExecutionContext) extends BackendController(cc) {
```

Comment:
if we can't move the implicit up we can make the bracket placement consistent so the ) is on the controllerComponents line?

Reply:
Done